src/cmd/build.rs

```diff
@@ -396,6 +416,7 @@ fn do_optimization(
                 err
             )
         })?;
+    // TODO
```

Comment:
Is this a leftover, or are you missing some functionality here?

Reply:
Good catch, thx! I removed it and added a commit which makes our CI report this in the future (as we do in ink!).